recipes/uncrustify/all/test_package/conanfile.py

```diff
@@ -0,0 +1,10 @@
+from conans import ConanFile, tools
+import sys
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
+
+    def test(self):
+        if not tools.cross_building(self.settings):
```

Comment:
```suggestion
        if not tools.cross_building(self):
```

Use `self`only. self.settings is autodated